sematic/ui/src/base64ArrayBuffer.js

```diff
@@ -0,0 +1,64 @@
+// From https://gist.github.com/jonleighton/958841
```

Comment:
Premature optimization?

Reply:
Some background:
https://stackoverflow.com/questions/9267899/arraybuffer-to-base64-encoded-string#answers-header

I ran the benchmark; this non-native approach seems quicker.